community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/fusion/FusionSchemaIndexProvider.java

```diff
@@ -80,29 +78,46 @@ public IndexAccessor getOnlineAccessor( long indexId, IndexDescriptor descriptor
     @Override
     public String getPopulationFailure( long indexId ) throws IllegalStateException
     {
-        String nativeFailure = nativeProvider.getPopulationFailure( indexId );
-        String luceneFailure = luceneProvider.getPopulationFailure( indexId );
-        if ( nativeFailure != null )
+        String nativeFailure = null;
+        try
         {
-            return luceneFailure == null ? nativeFailure : nativeFailure + " and " + luceneFailure;
+            nativeFailure = nativeProvider.getPopulationFailure( indexId );
         }
-        else
+        catch ( IllegalStateException e )
+        {   // Just catch
+        }
+        String luceneFailure = null;
+        try
         {
-            return luceneFailure;
+            luceneFailure = luceneProvider.getPopulationFailure( indexId );
+        }
+        catch ( IllegalStateException e )
+        {   // Just catch
         }
+
+        if ( nativeFailure != null || luceneFailure != null )
```

Comment:
I think we want to check `null` explicitly. Empty string means something else in this case.